File: php/Garage.php
<?php

require_once 'Car.php';
require_once 'Motorcycle.php';

class Garage {
    private $name;
    private $sizeArea;
    private $listOfCar;
    private $listOfMotorcycle;

    // Constructor
    public function __construct($name, $sizeArea) {
        $this->name = $name;
        $this->sizeArea = $sizeArea;
        $this->listOfCar = array();
        $this->listOfMotorcycle = array();
    }

    // Getters
    public function getName() {
        return $this->name;
    }

    public function getSizeArea() {
        return $this->sizeArea;
    }

    public function getCars() {
        return $this->listOfCar;
    }

    public function getMotorcycles() {
        return $this->listOfMotorcycle;
    }

    // Setters
    public function setName($name) {
        $this->name = $name;
    }

    public function setSizeArea($sizeArea) {
        $this->sizeArea = $sizeArea;
    }

    public function addCar($car) {
        $this->listOfCar[] = $car;
    }

    public function addMotorcycle($motorcycle) {
        $this->listOfMotorcycle[] = $motorcycle;
    }

    public function print() {
        echo $this->name . "<br/>";

        echo "<br/>List of number plate of car<br/>";
        foreach ($this->listOfCar as $car) {
            echo $car->getNumberPlate() . "<br/>";
        }

        echo "<br/>List of number plate of motorcycle<br/>";
        foreach ($this->listOfMotorcycle as $motorcycle) {
            echo $motorcycle->getNumberPlate() . "<br/>";
        }

        echo "<br/>";
    }
}

File: python/Garage.py
class Garage:
    def __init__(self, name, sizeArea):
        self._name = name
        self._sizeArea = sizeArea
        self._listOfCar = []
        self._listOfMotorcycle = []

    # Getters
    def getName(self):
        return self._name

    def getSizeArea(self):
        return self._sizeArea

    def getCars(self):
        return self._listOfCar

    def getMotorcycles(self):
        return self._listOfMotorcycle

    # Setters
    def setName(self, name):
        self._name = name

    def setSizeArea(self, sizeArea):
        self._sizeArea = sizeArea

    def addCar(self, car):
        self._listOfCar.append(car)

    def addMotorcycle(self, motorcycle):
        self._listOfMotorcycle.append(motorcycle)

    def printGarage(self):
        print(self._name)

        print("\nList of number plates of cars")
        for car in self._listOfCar:
            print(car.getNumberPlate())

        print("\nList of number plates of motorcycles")
        for motorcycle in self._listOfMotorcycle:
            print(motorcycle.getNumberPlate())

        print()
